include/baresip.h

```diff
@@ -831,7 +832,7 @@ void uag_enable_sip_trace(bool enable);
 int  uag_reset_transp(bool reg, bool reinvite);
 void uag_set_sub_handler(sip_msg_h *subh);
 int  uag_set_extra_params(const char *eprm);
-struct ua   *uag_find(const struct pl *cuser);
+struct ua   *uag_find(const struct sip_msg *msg);
```

Comment:
I suggest adding a new function:

`uag_find_msg(const struct sip_msg *msg);`

and leave the current one unchanged.

this way we dont break the API :)

Reply:
Ok, I but then I'll change the first commit. This is better to review. uag_find() will be unchanged and uag_find_msg() added.

Note: uag_find() will be un-used also.